resources/core/values.yaml

```diff
@@ -24,6 +24,8 @@ global:
   istio:
     tls:
       secretName: istio-ingress-certs
+  ectdBackupABS:
```

Comment:
I guess it should be in cluster.yaml file. Please check.

Reply:
discussed, this should be defined here, because `cluster.yaml` is used only for cluster overrides :)